src/counters.c

```diff
@@ -1566,5 +1646,6 @@ void StatsRegisterTests(void)
     UtRegisterTest("StatsTestUpdateGlobalCounter10",
                    StatsTestUpdateGlobalCounter10);
     UtRegisterTest("StatsTestCounterValues11", StatsTestCounterValues11);
+    UtRegisterTest("StatsTestQSortStatsTable", StatsTestQSortStatsTable);
```

Comment:
Per your question, yes, add a suffix to the test name.

IMO, it's ok to renumber the test cases since `01` no longer exists.

Reply:
Thanks, I'll change renumber all of them with my next PR, then.